### ISSUE TITLE
disable document signatures option is incorrect

### ISSUE THREAD

**Issue by steev** [ferrocene/ferrocene#788]
Hello! I've attempted to build ferrocene here on an arm64 host (Lenovo Thinkpad X13s), and when running the command

`OUTSIDE_FERROUS=1 CI=1 FERROCENE_HOST=aarch64-unknown-linux-gnu ./ferrocene/ci/configure.sh`

It spits out an error:

```
steev@finn:~/git/ferrocene$ OUTSIDE_FERROUS=1 CI=1 FERROCENE_HOST=aarch64-unknown-linux-gnu ./ferrocene/ci/configure.sh
configure: processing command line
configure:
configure: build.configure-args := ['--set', 'build.metrics', '--disable-manage-s ...
configure: build.metrics        := True
configure: dist.compression-profile := balanced
configure: rust.remap-debuginfo := True
configure: rust.debug-logging   := False
configure: rust.jemalloc        := True
configure: target.aarch64-unknown-nto-qnx710.cc := qcc
configure: target.aarch64-unknown-nto-qnx710.cxx := q++
configure: target.aarch64-unknown-nto-qnx710.ar := ntoaarch64-ar
configure: target.aarch64-unknown-nto-qnx710.profiler := False
configure: target.x86_64-pc-nto-qnx710.cc := qcc
configure: target.x86_64-pc-nto-qnx710.cxx := q++
configure: target.x86_64-pc-nto-qnx710.ar := ntox86_64-ar
configure: target.x86_64-pc-nto-qnx710.profiler := False
configure: rust.codegen-units-std := 1
configure: rust.verify-llvm-ir  := True
configure: target.wasm32-unknown-unknown.profiler := False
configure: ferrocene.channel    := beta
configure: ferrocene.traceability-matrix-mode := ci
configure: ferrocene.ignore-document-signatures := True
configure: build.submodules     := False
configure: build.cargo-native-static := True
configure: llvm.static-libstdcpp := True
configure: dist.compression-formats := ['xz']
configure: rust.debuginfo-level-std := 1
configure: rust.description     := Ferrocene by Ferrous Systems
configure: build.host           := ['aarch64-unknown-linux-gnu']
configure: build.target         := ['aarch64-unknown-linux-gnu']
configure: build.locked-deps    := True
configure: llvm.assertions      := True
configure: rust.debug-assertions := True
configure: build.sanitizers     := True
configure: rust.codegen-backends := ['llvm']
configure: build.extended       := True
configure: build.tools          := ['rustdoc', 'cargo', 'llvm-tools', 'rustfmt',  ...
configure: build.profiler       := True
configure: rust.lld             := True
configure: rust.channel         := stable
configure: profile              := dist
Traceback (most recent call last):
  File "/home/steev/git/ferrocene/./src/bootstrap/configure.py", line 561, in <module>
    section_order, sections, targets = parse_args(sys.argv[1:])
                                       ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/steev/git/ferrocene/./src/bootstrap/configure.py", line 277, in parse_args
    return parse_example_config(known_args, config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/steev/git/ferrocene/./src/bootstrap/configure.py", line 436, in parse_example_config
    configure_file(sections, top_level_keys, targets, config)
  File "/home/steev/git/ferrocene/./src/bootstrap/configure.py", line 512, in configure_file
    configure_section(sections[section_key], section_config)
  File "/home/steev/git/ferrocene/./src/bootstrap/configure.py", line 488, in configure_section
    raise RuntimeError("failed to find config line for {}".format(key))
RuntimeError: failed to find config line for ignore-document-signatures
```

I believe that the option may have changed, and instead of `ignore-document-signatures=true` the option should be `document-signatures=disabled` And I've got a local modification of the following

```diff
 if ! is_internal; then
-    add --set ferrocene.ignore-document-signatures=true
+    add --set ferrocene.document-signatures=disabled
 fi
```

And this does seem to work
```
steev@finn:~/git/ferrocene$ OUTSIDE_FERROUS=1 CI=1 FERROCENE_HOST=aarch64-unknown-linux-gnu ./ferrocene/ci/configure.sh
configure: processing command line
configure:
configure: build.configure-args := ['--set', 'build.metrics', '--disable-manage-s ...
configure: build.metrics        := True
configure: dist.compression-profile := balanced
configure: rust.remap-debuginfo := True
configure: rust.debug-logging   := False
configure: rust.jemalloc        := True
configure: target.aarch64-unknown-nto-qnx710.cc := qcc
configure: target.aarch64-unknown-nto-qnx710.cxx := q++
configure: target.aarch64-unknown-nto-qnx710.ar := ntoaarch64-ar
configure: target.aarch64-unknown-nto-qnx710.profiler := False
configure: target.x86_64-pc-nto-qnx710.cc := qcc
configure: target.x86_64-pc-nto-qnx710.cxx := q++
configure: target.x86_64-pc-nto-qnx710.ar := ntox86_64-ar
configure: target.x86_64-pc-nto-qnx710.profiler := False
configure: rust.codegen-units-std := 1
configure: rust.verify-llvm-ir  := True
configure: target.wasm32-unknown-unknown.profiler := False
configure: ferrocene.channel    := beta
configure: ferrocene.traceability-matrix-mode := ci
configure: ferrocene.document-signatures := disabled
configure: build.submodules     := False
configure: build.cargo-native-static := True
configure: llvm.static-libstdcpp := True
configure: dist.compression-formats := ['xz']
configure: rust.debuginfo-level-std := 1
configure: rust.description     := Ferrocene by Ferrous Systems
configure: build.host           := ['aarch64-unknown-linux-gnu']
configure: build.target         := ['aarch64-unknown-linux-gnu']
configure: build.locked-deps    := True
configure: llvm.assertions      := True
configure: rust.debug-assertions := True
configure: build.sanitizers     := True
configure: rust.codegen-backends := ['llvm']
configure: build.extended       := True
configure: build.tools          := ['rustdoc', 'cargo', 'llvm-tools', 'rustfmt',  ...
configure: build.profiler       := True
configure: rust.lld             := True
configure: rust.channel         := stable
configure: profile              := dist
configure:
configure: writing `config.toml` in current directory
configure:
configure: run `python /home/steev/git/ferrocene/x.py --help`
```

I'd submit it as a pull request, but I'm not sure if it's just something that was overlooked or if the option should actually be ignore-documment-signatures and the change should be elsewhere.

I also tested this against main as well as release/1.79

**Comment by Hoverbear** [comment]
Thanks for catching this, I'm going to take a look.

**Comment by steev** [comment]
This was fixed by #788 so I'm gonna close, thank you!
